addons/additional-stats/studioFollowers.js

```diff
@@ -0,0 +1,14 @@
+export default async function ({ addon, console, msg }) {
+    let studio = await (await fetch(`https://api.scratch.mit.edu/studios/${addon.tab.redux.state.studio.id}`)).json();
+
+    const timeSinceCreated = (Date.now() - Date.parse(studio.history.created)) / 86400000;
+    const followersPerDay = studio.stats.followers / timeSinceCreated;
```

Comment:
This can be larger than the total number of followers if the studio was created on the same day.